src/vs/base/browser/ui/dropdown/dropdown.ts

```diff
@@ -248,19 +252,24 @@ export class DropdownMenu extends BaseDropdown {
 			getActionItem: action => this.menuOptions && this.menuOptions.actionItemProvider ? this.menuOptions.actionItemProvider(action) : null,
 			getKeyBinding: action => this.menuOptions && this.menuOptions.getKeyBinding ? this.menuOptions.getKeyBinding(action) : null,
 			getMenuClassName: () => this.menuClassName,
-			onHide: () => this.onHide(),
+			onHide: () => this.hideMenu.schedule(),
 			actionRunner: this.menuOptions ? this.menuOptions.actionRunner : null,
 			anchorAlignment: this.menuOptions.anchorAlignment
 		});
+
+		setTimeout(() => this._contextMenuProvider.hideContextMenu(), 5000);
```

Comment:
This seems like debug code forgotten to be removed?

Reply:
yep testing the scenario I described above

Reply:
@sbatten that should then be removed